closed/get_j9_source.sh

```diff
@@ -233,6 +233,10 @@ if [ -n "${git_urls[$openjceplus_source]}" ] ; then
 	mkdir -p src/main/openjceplus/share/classes
 	cp -r src/main/java/* src/main/openjceplus/share/classes/
 
+	# Create a folder for the legal files as well.
+	mkdir -p src/main/openjceplus/share/legal
+	cp LICENSE src/main/openjceplus/share/legal/OPENJCEPLUS_LICENSE
```

Comment:
I think a better place for this is in `closed/custom/modules/openjceplus/Copy.gmk` which would say something like:
```
include $(TOPDIR)/closed/CopySupport.gmk

$(call openj9_copy_files,, \
	$(OPENJCEPLUS_TOPDIR)/LICENSE \
	$(LEGAL_DST_DIR)/OPENJCEPLUS_LICENSE)
```

Reply:
Sorry, I meant to refer to `closed/make/modules/openjceplus/Copy.gmk`.